src/main/java/com/liefery/android/stop_badge/StopBadgeView.java

```diff
@@ -180,7 +180,7 @@ protected void onLayout(
         if ( cache == null ) {
             int width = right - left;
             int height = bottom - top;
-            int size = Math.min( width, height );
+            int size = Math.max( width, height );
```

Comment:
This is the size of the circle shape. The actual size of the resulting `Bitmap` will be bigger when shadows are added.